cms/djangoapps/contentstore/views/course.py

```diff
@@ -410,15 +411,80 @@ def course_filter(course_summary):
             return False
 
         return has_studio_read_access(request.user, course_summary.id)
+
+    enable_home_page_api_v2 = settings.FEATURES["ENABLE_HOME_PAGE_COURSE_API_V2"]
+
     if org is not None:
         courses_summary = [] if org == '' else CourseOverview.get_all_courses(orgs=[org])
```

Comment:
If I'm reading this correctly, if edx.org uses `ENABLE_GLOBAL_STAFF_OPTIMIZATION` this would mean that they already read the courses from `CourseOverviews` instead of reading from `modulestore().get_course_summaries()`.

This would make the change even lower risk for them to turn on.

Reply:
Yes, you're correct. For the record, this is working if we turn on API V2, but try using API V1. This can be seen by testing locally or by checking the unittests.